var/spack/repos/builtin/packages/r-sf/package.py

```diff
@@ -29,4 +29,4 @@ class RSf(RPackage):
     depends_on('gdal@2.0.1:')
     depends_on('geos@3.4.0:')
     depends_on('proj@4.8.0:5', when='@:0.7-3')
-    depends_on('proj@4.8.0:', when='@0.7-4:')
+    depends_on('proj@4.8.0:6.3.2', when='@0.7-4:')
```

Comment:
```suggestion
    depends_on('proj@4.8.0:6', when='@0.7-4:')
```

Reply:
why we cannot use `6.3.2`?

Reply:
In case a bugfix release like 6.3.3 comes out. What we want is <= 6, not <= 6.3.2.

Reply:
We found it will make build error by using `depends_on('proj@4.8.0:6', when='@0.7-4:')`
`proj@6` can not be download. 
Shall we use `depends_on('proj@4.8.0:6.0.0', when='@0.7-4:')` ?
@adamjstewart

Reply:
Hmm, can you try `6.999`?

Reply:
> Hmm, can you try `6.999`?

`6.999` will use `6.3.2`, but you were not recommend this version?

I found all versions of `proj` has a RC version too:
```
[root@Estuary-CentOS8 spack_source]# spack versions proj
==> Safe versions (already checksummed):
  7.1.0  7.0.1  7.0.0  6.3.2  6.3.1  6.2.0  6.1.0  6.0.0  5.2.0  5.1.0  5.0.1  4.9.2  4.9.1  4.8.0  4.7.0  4.6.1
==> Remote versions (not yet checksummed):
  7.2.1RC1  7.2.0RC1  7.1.1     7.0.1RC1  6.3.2RC1  6.3.0     6.2.0RC1  6.1.0RC3  6.0.0RC4  6.0.0RC1  5.1.0RC2  5.0.0RC6  5.0.0RC3  5.0.0     4.9.3RC1  4.9.2RC1  4.9.1RC2  4.9.0RC2  4.5.0  4.4.7  4.4.4  4.4.1
  7.2.1     7.2.0     7.1.0RC2  7.0.0RC2  6.3.1RC1  6.2.1RC1  6.1.1RC1  6.1.0RC2  6.0.0RC3  5.2.0RC2  5.1.0RC1  5.0.0RC5  5.0.0RC2  4.9.3RC3  4.9.3     4.9.1RC4  4.9.1RC1  4.9.0RC1  4.4.9  4.4.6  4.4.3  4.4.0
  7.2.0RC2  7.1.1RC1  7.1.0RC1  7.0.0RC1  6.3.0RC1  6.2.1     6.1.1     6.1.0RC1  6.0.0RC2  5.2.0RC1  5.0.1RC1  5.0.0RC4  5.0.0RC1  4.9.3RC2  4.9.2RC2  4.9.1RC3  4.9.0b2   4.6.0     4.4.8  4.4.5  4.4.2
```